opm/material/fluidsystems/BlackOilFluidSystem.hpp

```diff
@@ -994,6 +994,41 @@ class BlackOil : public BaseFluidSystem<Scalar, BlackOil<Scalar> >
     }
 
     /*!
+     * \brief Returns the bubble point pressure $P_b$ using the current gas saturation in the fluid phase (Rs)
+     */
+    template <class FluidState, class LhsEval = typename FluidState::Scalar>
+    static LhsEval bubblePointPressure(const FluidState& fluidState,
+                                       unsigned regionIdx)
+    {
+        assert(0 <= regionIdx && regionIdx <= numRegions());
+
+        typedef Opm::MathToolbox<typename FluidState::Scalar> FsToolbox;
+
+        const auto& T = FsToolbox::template decay<LhsEval>(fluidState.temperature(oilPhaseIdx));
+        const auto& Rs = FsToolbox::template decay<LhsEval>(fluidState.Rs());
```

Comment:
it is better to use `getRs_(fluidState, regionIdx)` here (cf, https://github.com/OPM/opm-material/blob/master/opm/material/fluidsystems/BlackOilFluidSystem.hpp#L53 ), because this also works for fluid states that do not feature a `Rs()` method.